src/guide.jsx

```diff
@@ -7,27 +7,28 @@ export default class GuidePanel extends React.Component {
     constructor(props) {
         super(props)
         this.state = {
-            video: null
+            video: null,
         }
         this.client = io('http://localhost:9091');
-        this.client.on('now-playing', (data) => {
-            console.log('Received data: ' + data);
-        })
+        this.client.on('now-playing', this.handleUpdate);
+    }
+
+    handleUpdate = (data) => {
+        console.log('Received data: ');
+        console.log(data)
+        this.setState({ video: data });
+        console.log(this.state);
     }
 
 
     render() {
-        if (this.state.video == null) {
-            return (
-                <div>
-                    <p>Not connected</p>
-                </div>
-            )
-        }
-        return (
+        return ( this.state.video === null ? 
```

Comment:
What does this do?

Reply:
It's a [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator), basically if this.state.video is null return the first, otherwise return the second.

Reply:
neat